src/main/resources/application.properties

```diff
@@ -0,0 +1,8 @@
+spring.datasource.url=${JDBC_URL}?currentSchema=tododb
```

Comment:
Can't we use public schema? Why tododb as a schema? Schema is not a db in postgres